pkg/controller/directvolumemigration/descriptions.go

```diff
@@ -1,24 +1,28 @@
-	CreatePVProgressCRs:             "Creating direct volume migration progress CR to get progress percent and transfer rate",
-	CreateRsyncTransferPods:         "Creating rsync daemon pods on the destination cluster",
-	WaitForRsyncTransferPodsRunning: "Wait for the rsync daemon pod to be running",
-	CreateStunnelClientPods:         "Create stunnel client pods on the source cluster",
-	WaitForStunnelClientPodsRunning: "Waiting for the stunnel client pods to be running",
-	CreateRsyncClientPods:           "Create Rsync client pods",
-	WaitForRsyncClientPodsCompleted: "Wait for the rsync client pods to be completed",
-	DeleteRsyncResources:            "Delete all the resources created while running this migration",
-	MigrationFailed:                 "The migration attempt failed, please see errors for more details",
-	Completed:                       "Complete",
+	Created:                              "DVM CR has been created",
+	Started:                              "DVM Controller is configuring DVM CR",
+	Prepare:                              "DVM Controller is preparing the environment for volume migration. Currently it is non-operational",
+	CleanStaleRsyncResources:             "Cleaning up stale resources from previous migrations",
+	WaitForStaleRsyncResourcesTerminated: "Waiting for stale resources to terminate",
+	CreateDestinationNamespaces:          "Creating target namespaces",
+	DestinationNamespacesCreated:         "Checking if the target namespaces have been created, currently it is non-operational",
+	CreateDestinationPVCs:                "Creating PVCs in the target namespaces",
+	DestinationPVCsCreated:               "Checking whether the created PVCs are bound",
+	CreateRsyncRoute:                     "Creating one route for each namespace for Rsync on the target cluster",
+	CreateRsyncConfig:                    "Creating a config map and secrets on both the source and target clusters for Rsync configuration",
+	CreateStunnelConfig:                  "Creating a config map and secrets for Stunnel to connect to Rsync on the source and target clusters",
+	CreatePVProgressCRs:                  "Creating a Direct Volume Migration Progress CR to get progress percent and transfer rate",
+	CreateRsyncTransferPods:              "Creating Rsync daemon pods on the target cluster",
+	WaitForRsyncTransferPodsRunning:      "Waiting for the Rsync daemon pod to run",
+	CreateStunnelClientPods:              "Creating Stunnel client pods on the source cluster",
+	WaitForStunnelClientPodsRunning:      "Waiting for the Stunnel client pods to run",
+	CreateRsyncClientPods:                "Creating Rsync client pods",
+	WaitForRsyncClientPodsCompleted:      "Waiting for the Rsync client pods to be completed",
+	DeleteRsyncResources:                 "Deleting resources created by this migration",
+	WaitForRsyncResourcesTerminated:      "Waiting for resources to terminate",
+	Verification:                         "Verifying migration was successful",
```

Comment:
nit: how do we plan on doing this? is this for the future?

Reply:
+1 this is a future thing. I assume this would mean actually verifying source PVC matches destination PVC. It's not currently used.

Reply:
thanks!